pkg/security/module/module.go

```diff
@@ -167,7 +167,13 @@ func (m *Module) Start() error {
 	}
 
 	if m.config.SelfTestEnabled && m.selfTester != nil {
-		_ = m.RunSelfTest(true)
+		if triggerred, err := m.RunSelfTest(true); err != nil {
+			err = fmt.Errorf("failed to run self test: %s", err)
```

Comment:
Should we put this inside the if statement, otherwise it seems that we will send ~twice the same text line 176 ?

Reply:
This is part of the initial PR, but I think the idea is `if triggered then return error, else then log`